src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/services/service-form/service-form.component.html

```diff
@@ -74,16 +74,23 @@
         <div class="form-group row"
              *ngIf="serviceForm.controls.service_type.value !== 'snmp-gateway'">
           <label class="cd-col-form-label"
-                 [ngClass]="{'required': ['mds', 'rgw', 'nfs', 'iscsi', 'ingress'].includes(serviceForm.controls.service_type.value)}"
+                 [ngClass]="{'required': ['mds', 'rgw', 'nfs', 'iscsi', 'nvmeof', 'ingress'].includes(serviceForm.controls.service_type.value)}"
                  for="service_id">
-            <span i18n>Id</span>
-            <cd-helper i18n>Used in the service name which is &lt;service_type.service_id&gt;</cd-helper>
+            <span i18n>Service Name</span>
           </label>
           <div class="cd-col-form-input">
-            <input id="service_id"
-                   class="form-control"
-                   type="text"
-                   formControlName="service_id">
+            <div class="input-group">
+              <span class="input-group-text"
+                    *ngIf="serviceForm.controls.service_type.value"
+                    for="userId"
+                    i18n>{{serviceForm.controls.service_type.value}}.
+              </span>
+              <input id="service_id"
+                     class="form-control"
+                     type="text"
+                     formControlName="service_id">
+            </div>
+            <cd-help-text i18n>The name of the service which is &lt;service_type.service_id&gt;</cd-help-text>
```

Comment:
```suggestion
 ```
 
 Is it needed anymore? After changing the field name to `Service Name` and having the service type prefixed by default, I think that's enough

Reply:
yeah! will remove, thanks!

Reply:
Done @Pegonzal